compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -935,6 +935,123 @@ OMR::ARM64::TreeEvaluator::vxorEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return inlineVectorBinaryOp(node, cg, xorOp);
    }
 
+TR::Register *
+OMR::ARM64::TreeEvaluator::vminmaxInt64Helper(TR::Node *node, bool isMax, TR::CodeGenerator *cg)
```

Comment:
How about making this helper function a static function instead of a member function of TreeEvaluator if it is called only by vmin/vmaxEvaluator()?

Reply:
Change the helper function to a static function.